tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -114,4 +114,23 @@ class WskWebActionsTests
             authorizedResponse.statusCode shouldBe 200
             authorizedResponse.body().asString() shouldBe namespace
     }
+
+    it should "ensure that CORS header is preserved" in withAssetCleaner(wskprops) {
```

Comment:
@rabbah, please let me know if this is the correct test you wanted to be included in WskWebActionTests.